srcdeps-maven-quickstarts/srcdeps-mvn-git-parent-quickstart/pom.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2015-2016 Maven Source Dependencies
+    Plugin contributors as indicated by the @author tags.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.l2x6.maven.srcdeps.itest</groupId>
+    <artifactId>srcdeps-test-artifact</artifactId>
+    <version>0.0.2-SRC-revision-3d00c2a91af593c01c9439cb16cb5f52d2ddbcf8</version>
+  </parent>
+
+  <groupId>org.srcdeps.mvn.quickstarts.srcdeps-mvn-git-parent-quickstart</groupId>
+  <artifactId>srcdeps-mvn-git-parent-quickstart</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <packaging>jar</packaging>
+  <description>A project using a source revision of another project as its parent</description>
+
+  <dependencies>
+    <dependency>
+        <groupId>org.l2x6.maven.srcdeps.itest</groupId>
+        <artifactId>srcdeps-test-artifact-api</artifactId>
+        <!-- Managed in parent -->
+    </dependency>
+
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.12</version>
+      <scope>test</scope>
+    </dependency>
+
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <systemProperties>
+            <srcdeps.testString>${srcdeps.testString}</srcdeps.testString><!-- defined in parent -->
```

Comment:
Hehe, that's the essence of this test: we use srcdeps-test-artifact as a parent of this quickstart https://github.com/srcdeps/srcdeps-maven/pull/2/files/329de610163c35f8c7b57ad8085e2facbfb0ceb7#diff-6dff057a5529a50a82f04bee58ff17b5R25

and `srcdeps.testString` is defined there in the referenced revision of srcdeps-test-artifact: https://github.com/srcdeps/srcdeps-test-artifact/commit/3d00c2a91af593c01c9439cb16cb5f52d2ddbcf8

We do not define `srcdeps.testString` anywhere else. Hence unless the parent-child srcdep works, this test in the quickstart would fail https://github.com/srcdeps/srcdeps-maven/pull/2/commits/fe11ae55ac6c33930bd5eb347e72d9babd9bd187#diff-0b16c394cd7498a06482d4848c825559R38 and hence the whole quickstart build would fail.